.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+
+//! need to cleanup, also codespace ran out of space (8 -core 32GB RAM • 64GB works so far without losing space)
+//! has over an hour + upstart and needs further investigation overall
+
+{
+	"name": "UIPA DevBox",
+	"build": {
+	  "context": "..",
+	  "dockerfile": "../Dockerfile"
+	},
+	"runArgs": [
+	  "--init",
+	  "--privileged"
+	],
+	"forwardPorts": [8000, 5432, 9200],
+	// "postCreateCommand": "sudo usermod -aG docker $USER && sudo service docker start",
+	"postCreateCommand": "sudo usermod -aG docker $USER",
+	"postStartCommand": "bash cleanup_and_start.sh",
```

Comment:
Did you find that we needed to do this sort of cleanup and starting of Docker for some reason?

Reply:
I started a Codespace off the `main` branch and did a `docker info` - Docker should be installed and run by default.

```
@tyliec ➜ /workspaces/uipa (main) $ docker info
Client:
 Version:    26.1.3-1
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.14.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.27.1
    Path:     /usr/libexec/docker/cli-plugins/docker-compose
...
```

Not sure if you found some sort of timing issue or not here though